avm/res/digital-twins/digital-twins-instance/README.md

```diff
@@ -115,7 +115,7 @@ module digitalTwinsInstance 'br/public:avm/res/digital-twins/digital-twins-insta
       endpointUri: '<endpointUri>'
       entityPath: '<entityPath>'
       managedIdentities: {
-        userAssignedResourceId: '<userAssignedResourceId>'
+        userAssignedResourceIds: '<userAssignedResourceIds>'
```

Comment:
@ChrisSidebotham @AlexanderSehr is this PR still relevant or was that covered by #1400 ? If so can we close it?

Reply:
@eriqua, I think this is a now 😄